### ISSUE TITLE
Remove backticks from `ddev mailpit` in `ddev describe` info text

### ISSUE THREAD

**Issue by gitressa** [ddev/ddev#6276]
### Preliminary checklist

- [X] I am using the latest stable version of DDEV (see [upgrade guide](https://ddev.readthedocs.io/en/stable/users/install/ddev-upgrade/))
- [X] I have searched [existing issues](https://github.com/ddev/ddev/issues)
- [X] I have checked the [troubleshooting guide](https://ddev.readthedocs.io/en/stable/users/usage/troubleshooting/)
- [X] I have run `ddev debug test` to include output below

### Problem

Under `ddev describe` the Mailpit help text is shown as `` `ddev mailpit` ``, using backticks, which don't really add anything.

```
$ ddev describe
┌─────────────────────────────────────────────────────────────────────────────────────────┐
│ Project: solr-test ~/dev/solr-test https://solr-test.ddev.site                          │
│ Docker platform: linux-docker                                                           │
│ Router: traefik                                                                         │
├──────────┬──────┬─────────────────────────────────────────────────┬─────────────────────┤
│ SERVICE  │ STAT │ URL/PORT                                        │ INFO                │
├──────────┼──────┼─────────────────────────────────────────────────┼─────────────────────┤
[... ]
├──────────┼──────┼─────────────────────────────────────────────────┼─────────────────────┤
│ Mailpit  │      │ Mailpit: https://solr-test.ddev.site:8026       │                     │
│          │      │ `ddev mailpit`                                  │                     │
├──────────┼──────┼─────────────────────────────────────────────────┼─────────────────────┤
```

### Solution

Update `` `ddev mailpit` `` and remove the backticks, to be just `ddev mailpit`.

**Comment by stasadev** [comment]
I think we have these back ticks to make it clear that this is a command that you can run and not something else.

We actually use all of these in the DDEV output, depending on the developer's preference 🙂 :
- `
- "
- '

**Comment by gitressa** [comment]
Thanks for a fast response @stasadev, it's great.

But in my opinion, the backticks (or any other indication) are not needed. From the context, the user will understand the meaning. If anything (if help is needed) it could be something like this?

> Launch: ddev mailpit

**Comment by stasadev** [comment]
This is a good suggestion, I like it. Feel free to contribute it with PR.

**Comment by gitressa** [comment]
Thanks! I created a PR. 

PS. Perhaps the PR template could be updated, and `Fixes #<issue number>.` added under `## The Issue`, so that it helps the user connect the issue and the PR which solves it? Perhaps something like this?:

```
[...]
## The Issue

Fixes #<issue number>.

## How This PR Solves The Issue
[...]
```

**Comment by stasadev** [comment]
Thank you @gitressa for attention to such details.

I always use the list syntax for issue references, like `- #6280`, so it will be expanded by GitHub, which is more readable:

---
Example with `Fixes #6280`

Fixes #6280
---

Example with `- #6280`

- #6280

**Comment by gitressa** [comment]
Ah yes, that's even better, thanks! I very much appreciate your efforts here with DDEV @stasadev -- you have been an extremely efficient and knowledgeable addition to the team, so kudos to @rfay for choosing you.

Thanks for creating a PR about updating the PR issue template, and then using it as an example -- very [meta](https://english.stackexchange.com/questions/508638/what-does-very-meta-mean) :)
